Resources/views/Form/jquery_layout.html.twig

```diff
@@ -656,6 +656,10 @@
     <script type="text/javascript">
     jQuery(document).ready(function($) {
         $field = $('#{{ id }}');
+        
+        {% if configs.placeholder is defined %}
+            {% set configs = {'placeholder': configs.placeholder|trans} + configs %}
```

Comment:
don't work.

> Unexpected token "punctuation" of value "." ("end of statement block" expected) in GenemuFormBundle:Form:jquery_layout.html.twig at line 661.

The original method works well
`{% set configs = {'placeholder': configs.placeholder|trans} + configs %}`
or mine
`{% set configs = configs|merge({"placeholder": configs.placeholder|trans}) %}`